src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml

```diff
@@ -786,7 +786,7 @@
     <sys:String x:Key="Ques_Delete_All_Installations">Vuoi eliminare tutti i file di installazione?</sys:String>
     <sys:String x:Key="Ques_Delete_Installation">Vuoi eliminare {0}?</sys:String>
     <sys:String x:Key="Ques_Delete_Installation_Package">Vuoi eliminare {0}?</sys:String>
-    <sys:String x:Key="Ques_Install_iSlide">PlaceHolder</sys:String>
+    <sys:String x:Key="Ques_Install_iSlide">Segnaposto</sys:String>
```

Comment:
PlaceHolder is not required to translate.

Reply:
Done. 
Restored PlaceHolder.